src/liquid/components/place_item.html

```diff
@@ -1,88 +1,103 @@
-        {% elsif place.validationErrors != empty %}is-warning
+        {% if place.state == 'failure' %}error-bg
+        {% elsif place.state == 'success' %}success-bg
+        {% elsif place.validationErrors != empty %}warning-bg
+        {% elsif place.state == 'staged' %}staging-bg
+        {% elsif place.state == 'in_progress' %}striped-bg
         {% endif %}
         {% if place.uploadError %}has-tooltip-arrow has-tooltipl-multiline{% endif %}
         {% endcapture %}
-        <span class="tag is-info {{ tag_class }}" {% if place.uploadError
-            %}data-tooltip="{{ place.uploadError | escape }}" {% endif %}>
-            {{ tag_text }}
-        </span>
+        <div class="status badge {{ tag_class }}" {% if place.uploadError
+            %}title="{{ place.uploadError | escape }}" data-toggle="tooltip" data-placement="top" {% endif %}>
+            {% if place.state == 'in_progress' %} uploading {% else %} {{ tag_text }} {%endif%}
+        </div>
     </td>
 
     <td>
         {% if place.creationDetails.password %}
-        {% capture explanation %}Username: {{ place.creationDetails.username }} Password: {{ place.creationDetails.password }}{% endcapture %}
-        <span class="has-tooltip-arrow has-tooltipl-multiline material-symbols-outlined"
-            data-tooltip="{{ explanation | escape }}"
+        {% capture explanation %}Username: {{ place.creationDetails.username }} Password: {{
+        place.creationDetails.password }}{% endcapture %}
+        <span class="action-link"
+            title="Click to copy {{ explanation | escape }}" data-toggle="tooltip" data-placement="top"
             onclick="navigator.clipboard.writeText(`{{ explanation | escape }}`);">
-            key
+            <img class="action-link" src="public/icons/password.svg" alt="password">
         </span>
         {% else %}
-        <a href="/place/edit/{{place.id}}" data-tooltip="Edit">
-            <span class="material-symbols-outlined">edit</span>
+        <a class="action-link" href="/place/edit/{{place.id}}" data-toggle="tooltip" data-placement="bottom" title="Edit">
+            <img src="public/icons/edit.svg" alt="Edit">
         </a>
-        <a hx-post="/place/refresh/{{place.id}}" data-tooltip="Refresh" hx-target="this" hx-swap="none">
-            <span class="material-symbols-outlined">refresh</span>
+        <a class="action-link" hx-post="/place/refresh/{{place.id}}" data-toggle="tooltip" data-placement="bottom"
+        title="Refresh" hx-target="this" hx-swap="none">
+            <img src="public/icons/refresh.svg" alt="refresh">
         </a>
         {% if place.validationErrors == empty %}
-        <a hx-post="/place/upload/{{place.id}}" hx-target="this" hx-swap="none" data-tooltip="Upload">
-            <span class="material-symbols-outlined">upload</span>
+        <a class="action-link" hx-post="/place/upload/{{place.id}}" hx-target="this" hx-swap="none show:window:top" 
```

Comment:
Not sure if we really want to scroll way from an item we explicitly clicked an action on

Reply:
agreed will revert